boards/arm/cyclonev_socdk/Kconfig.defconfig

```diff
@@ -14,6 +14,11 @@ config I2C_DW_CLOCK_SPEED
 	default 200
 endif # I2C_DW
 
+if EEPROM
```

Comment:
This seems unrelated to the ethernet driver to me

Reply:
Resolved